Library/Formula/mpd.rb

```diff
@@ -116,4 +119,11 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    pid = fork { exec "#{bin}/mpd --stdout --no-daemon --no-config" }
+    sleep 2
+    Process.kill("SIGINT", pid)
+    Process.wait(pid)
+  end
```

Comment:
Should do. If it doesn't launch correctly, say because it can't find the correct linked `dylib` it shouldn't start the process, and there shouldn't be anything to kill. I'll revert my `icu4c` tonight and then install `mpd` again and then upgrade `icu4c` and run the test and see if it blows up as intended.